apps/wallet-ui/package.json

```diff
@@ -10,7 +10,9 @@
     "fix": "biome check ./src --fix"
   },
   "dependencies": {
+    "@tanstack/react-query": "^5.48.0",
     "next": "14.2.4",
+    "pino-pretty": "^11.2.1",
```

Comment:
definitely not a dependency, if this is shouting at you in logs check playground how we disabled it in `next config`